ABOS/ASFS/lftp_sync.sh

```diff
@@ -19,6 +18,22 @@ sync_files() {
     lftp -e "open -u $FTP_USER,$FTP_PASSWORD $FTP_SOURCE; mirror -e --parallel=10 $FTP_EXTRA_OPTS -vvv --log=$lftp_log_file $FTP_DIR $ABOS_SOFS_WIP_DIR_FTP/; quit"
 }
 
+# copy files to incoming dir with the correct permissions
+# $1 - subdirectory to copy to, relative to $INCOMING_DIR
+# "$@" - file(s) to copy
+copy_to_incoming() {
+    local incoming_dir="$INCOMING_DIR/$1" ; shift
+
+    for file; do
+        local file_basename=$(basename $file)
+        local tmp_file=$(mktemp -t "${file_basename}.XXXXXX")
+        cp $file $tmp_file && \
```

Comment:
The cp/chgrp/chmod chain can be condensed into a single install command, e.g. `install -g projectofficer -m 0664 $file $tmp_file`

Reply:
Thanks for the tip. I've updated the commit.